general/development/process/testing/guide.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Testing instructions guide
+tags:
+  - ' Processes'
+---
+This page has suggestions for developers on how to write good testing instructions for the weekly [testing of integrated issues](./integrated-issues).
+
+We recommend that you:
+
+1. Number the steps in your test, and make use of sub-lists.
+2. Only put one action (preparation or validation) on each line - A step should only define a unique operation.
+3. Promote test validations - **Confirm**, **Verify** or **Ensure** - should be in **bold** so that they are easily identifiable.
+4. Make use of the [Jira Markdown formatting](https://tracker.moodle.org/secure/WikiRendererHelpAction.jspa?section=all).
+
+In addition, the following items may be included:
+
+1. Setup requirements. If so, explain them with detailed information and/or provide a link to the documentation e.g. <https://docs.moodle.org/en/OAuth_2_services>.
+2. Whether testing involves git/shell interaction, SQL operations... (commands in general), don't assume the tester knows how to perform that stuff. Instead, add them explicitly to the instructions.
```

Comment:
They also recommend not using `...` (ellipsis) except under certain circumstances:

- https://docs.microsoft.com/en-us/style-guide/punctuation/ellipses